Sources/App/Setup/configure.swift

```diff
@@ -67,12 +66,7 @@ public func configure(
     services.register(OffsetPaginatorConfig.current)
 
     // MARK: Leaf tags
-
-    services.register { container -> LeafTagConfig in
-        var config = LeafTagConfig.default()
-        try leafTags(config: &config, container)
-        return config
-    }
+    // Look at boot.swift
```

Comment:
We're registering leaf tags in `Setup/leafTags.swift` (https://github.com/nodes-vapor/template/blob/feature/adapt-to-submissions/Sources/App/Setup/leafTags.swift) so this function needs to be called. Also, is it needed to do it through `boot.swift` still? I believe we did that due to the (now outdated) `MutableLeafTagContainer`.

Reply:
do you have an example of how this should be done?